portal-ui/src/screens/Console/Tenants/TenantDetails/TenantDetails.tsx

```diff
@@ -133,12 +140,50 @@ const TenantDetails = ({ classes, match }: ITenantDetailsProps) => {
     }
   };
 
+  const loadInfo = () => {
+    const tenantName = match.params["tenantName"];
+
+    setLoading(true);
+
+    api
+      .invoke("GET", `/api/v1/mkube/tenants/${tenantName}`)
+      .then((res: ITenant) => {
+        const total = res.volume_count * res.volume_size;
+
+        setCapacity(total);
+        setZoneCount(res.zone_count);
+        setVolumes(res.volume_count);
+        setInstances(res.instance_count);
+        const resZones = res.zones !== null ? res.zones : [];
```

Comment:
When there are no zones, res.zones is `undefined` in res variable, hence the object is not iterable and the zones doesn't get applied. Then with the validation of zones as null the modal box doesn't open

```suggestion
        const resZones = !res.zones ? [] : res.zones;
```